server/api.js

```diff
@@ -120,18 +125,24 @@ function getPosts(nums, isOP, cb) {
 			links = key + ':links';
 			m.hgetall(key);
 			m.hgetall(links);
+			m.get(key+':body')
```

Comment:
Not once, but TWICE

Reply:
I'll reflect on my shortcomings in the future.